src/com/actelion/research/gwt/core/JSReactor.java

```diff
@@ -23,6 +23,10 @@ public boolean setReactant(int no, JSMolecule reactant) {
   public JSMolecule[][] getProducts() {
     StereoMolecule[][] products = reactor.getProducts();
 
+    if (products == null || products.length == 0) {
+      return null;
+    }
```

Comment:
Shouldn't we return an empty array like the original code when we get an empty array?

Reply:
I would like but it is an array of array and if we return JSMolecule[] it is another type. I didn't know how to solve this issue.